windows_server_1709.json

```diff
@@ -168,11 +137,14 @@
     "disk_size": "61440",
     "disk_type_id": "1",
     "vhv_enable": "false",
-    "manually_download_iso_from": "https://my.visualstudio.com/downloads?pid=2347",
-    "iso_url": "https://software-download.microsoft.com/pr/en_windows_server_version_1709_x64_dvd_100090904.iso",
+    "manually_download_iso_from":
+      "https://my.visualstudio.com/downloads?pid=2347",
+    "iso_url":
+      "https://software-download.microsoft.com/pr/en_windows_server_version_1709_x64_dvd_100090904.iso",
```

Comment:
I'm getting 403 when trying to access the iso :-(

Reply:
Yes, these ISO's are not public available. Look at the "manually_download_iso_from" link above :-)
TL/DR you need a MSDN subscription to access the ISO. There is no eval ISO AFAIK.